Account & Project/3. Tutorials/2. 新舊計畫資料轉移.md

```diff
@@ -1 +1,114 @@
+---
+title: HowTo：新舊計畫資料轉移 | en
+tags: HowTo, EN
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# HowTo：Transfer data from old project to new project
+
+:::success
+<i class="fa fa-star" aria-hidden="true"></i> **Scenario：How to transfer data from old project to new project?**
+
+*Do you have question about transferring data? Let us show you how to connect TWCC cloud services step by step to answer your question !*
+:::
+
+If you already applied for new project and hope that the data from old project can be continuously used, please refer to this documentation. Transfer your data through **Cloud Object Storage (COS)**.
+
+
+**Sequence of data transferring：Old project COS** <i class="fa fa-forward" aria-hidden="true"></i> Download to **Localhost** <i class="fa fa-forward" aria-hidden="true"></i> Upload to **New project COS**
+
+
+According to your operating system, refer to [Localhost in Windows](#Localhost-in-Windows) or [Localhost in Linux](#Localhost-in-Linux).
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note：** Hyper File System (HFS) is attached to the user account, rather than project. Therefore, the data in HFS storage space does not need to transfer, and could still be accessed in the new project.
+:::
+
+
+## Localhost in Windows
+
+### Step 1. Back up the old project data to COS
+
+For those who use Virtual Compute Service (VCS), please refer to [this doc](https://www.twcc.ai/doc?page=backup). Back up the old project data in VCS Instance to COS.
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note：** 
+The default compute resource of Container Compute Service (CCS) and Taiwania 2 is Hyper File System (HFS), and personal data can be used cross projects. Therefore,  this step can be skipped.
```

Comment:
[SUG]

1. compute resource > storage
2. The default compute resource of Container Compute Service (CCS) and Taiwania 2 is Hyper File System (HFS), and personal data can be used cross projects. Therefore,  this step can be skipped. > 

Users of CCS and Taiwania 2 can skip this step because the default storage of the two services is Hyper File System (HFS).